From 5aebb07b5c9512155ce346bb71c0acbb04c26297 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Sun, 15 Sep 2024 08:42:06 +0300
Subject: [PATCH] [FSW-8694] fs1: arrange progress bits sequence

With this patch, arranged all progress bits
sequence with proper enum numbers.

Change-Id: Ib375d2e3fcfbed828bd9aa116fd5ff018d136538
---

diff --git a/arch/fs1/arc/progress_bits.h b/arch/fs1/arc/progress_bits.h
index 0c0f26d..e67104a 100644
--- a/arch/fs1/arc/progress_bits.h
+++ b/arch/fs1/arc/progress_bits.h
@@ -46,113 +46,112 @@
 	PROGRESS_INIT_SPI_VALID_MANIFEST_INDEX_FAIL = 36,
 	PROGRESS_INIT_EFUSE_INIT_FAIL = 37,
 	PROGRESS_INIT_EFUSE_SPI = 38,
-	PROGRESS_INIT_EFUSE_SPI_IMG_DESC_FAIL, /* TODO */
-	PROGRESS_INIT_EFUSE_SPI_EXCEEDS_MAX_SIZE, /* TODO */
-	PROGRESS_INIT_EFUSE_SPI_DMA_COPY_FAIL, /* TODO */
-	PROGRESS_INIT_EFUSE_SPI_MAGIC_FAIL, /* TODO */
-	PROGRESS_INIT_EFUSE_SPI_DESC_COUNT_FAIL, /* TODO */
-	PROGRESS_INIT_EFUSE_SPI_SLV_TIMEDOUT = 39,
-	PROGRESS_REVOKE_LOC_ASIC_EFUSE = 40,
-	PROGRESS_REVOKE_LOC_FAKE_FUSE = 41, /* TODO */
-	PROGRESS_KEY_LOC = 42,
-	PROGRESS_KEY_LOC_0 = 43,
-	PROGRESS_KEY_LOC_1 = 44,
-	PROGRESS_VER_LOC = 45,
-	PROGRESS_VER_LOC_0 = 46,
-	PROGRESS_VER_LOC_1 = 47,
-	PROGRESS_SLV_DIE_WAITING_ON_RECOVERY_IND = 48,
-	PROGRESS_SPI_MANIFEST_0_VERIFY_START = 49,
-	PROGRESS_SPI_MANIFEST_0_VERIFY_VERSION_FAIL = 50,
-	PROGRESS_SPI_MANIFEST_0_VERIFY_KEY_REVOKED = 51,
-	PROGRESS_SPI_MANIFEST_0_VERIFY_KEY_HASH_FAIL = 52,
-	PROGRESS_SPI_MANIFEST_0_VERIFY_AUTH_FAIL = 53,
-	PROGRESS_SPI_MANIFEST_0_VERIFY_SUCCESS = 54,
-	PROGRESS_SPI_MANIFEST_1_VERIFY_START = 55,
-	PROGRESS_SPI_MANIFEST_1_VERIFY_VERSION_FAIL = 56,
-	PROGRESS_SPI_MANIFEST_1_VERIFY_KEY_REVOKED = 57,
-	PROGRESS_SPI_MANIFEST_1_VERIFY_KEY_HASH_FAIL = 58,
-	PROGRESS_SPI_MANIFEST_1_VERIFY_AUTH_FAIL = 59,
-	PROGRESS_SPI_MANIFEST_1_VERIFY_SUCCESS = 60,
-	PROGRESS_BRINGUP_PCI_PRI = 61,
-	PROGRESS_BRINGUP_PCI_PRI_NO_TMG_DESC = 62,
-	PROGRESS_BRINGUP_PCI_PRI_LOADADDR_INVALID = 63,
-	PROGRESS_BRINGUP_PCI_PRI_STARTADDR_INVALID = 64,
-	PROGRESS_BRINGUP_PCI_PRI_TOO_BIG = 65,
-	PROGRESS_BRINGUP_PCI_PRI_CRC_FAIL = 66,
-	PROGRESS_BRINGUP_PCI_PRI_UNALIGNED_DEST_ADDR = 67,
-	PROGRESS_BRINGUP_PCI_PRI_INVALID_IMG_SIZE = 68,
-	PROGRESS_BRINGUP_PCI_PRI_UNALIGNED_IMG_SIZE = 69,
-	PROGRESS_BRINGUP_PCI_PRI_INVALID_FLASH_OFFSET = 70,
-	PROGRESS_BRINGUP_PCI_PRI_DMA_ERROR = 71,
-	PROGRESS_BRINGUP_PCI_PRI_DMA = 72,
-	PROGRESS_BRINGUP_PCI_PRI_KEY_REVOKED = 73,
-	PROGRESS_BRINGUP_PCI_PRI_KEY_AUTH_FAIL = 74,
-	PROGRESS_BRINGUP_PCI_PRI_SECURITY = 75,
-	PROGRESS_BRINGUP_PCI_SEC = 76,
-	PROGRESS_BRINGUP_PCI_SEC_NO_IMG_DESC = 77,
-	PROGRESS_BRINGUP_PCI_SEC_LOADADDR_INVALID = 78,
-	PROGRESS_BRINGUP_PCI_SEC_STARTADDR_INVALID = 79,
-	PROGRESS_BRINGUP_PCI_SEC_TOO_BIG = 80,
-	PROGRESS_BRINGUP_PCI_SEC_CRC_FAIL = 81,
-	PROGRESS_BRINGUP_PCI_SEC_UNALIGNED_DEST_ADDR = 82,
-	PROGRESS_BRINGUP_PCI_SEC_INVALID_IMG_SIZE = 83,
-	PROGRESS_BRINGUP_PCI_SEC_UNALIGNED_IMG_SIZE = 84,
-	PROGRESS_BRINGUP_PCI_SEC_INVALID_FLASH_OFFSET = 85,
-	PROGRESS_BRINGUP_PCI_SEC_DMA_ERROR = 86,
-	PROGRESS_BRINGUP_PCI_SEC_DMA = 87,
-	PROGRESS_BRINGUP_PCI_SEC_KEY_REVOKED = 88,
-	PROGRESS_BRINGUP_PCI_SEC_KEY_AUTH_FAIL = 89,
-	PROGRESS_BRINGUP_PCI_SEC_SECURITY = 90,
-	PROGRESS_BOOT_ROM_TPM = 91,
-	PROGRESS_BOOT_ROM_TPM_INIT_FAIL = 92,
-	PROGRESS_BOOT_ROM_TPM_STARTUP_FAIL = 93,
-	PROGRESS_BOOT_ROM_SVN_TPM_PCR_EXTEND_FAIL, /* TODO */
-	PROGRESS_BOOT_ROM_MANIFEST_TPM_PCR_EXTEND_FAIL = 94,
-	PROGRESS_BOOT_ROM_TPM_FSM_OUT_OF_SYNC, /* TODO */
-	PROGRESS_BOOT_ROM_TPM_FINISH = 95,
-	PROGRESS_PPBOOT_START = 96,
-	PROGRESS_PPBOOT_FINISH = 97,
-	PROGRESS_BOOT_ROM_IN_RECOVERY = 98,
-	PROGRESS_BOOT_ROM_IN_ROM_RECOVERY = 99,
-	PROGRESS_BOOT_ROM_PCIE_LINK_START = 100,
-	PROGRESS_BOOT_ROM_PCIE_LINK_INIT = 101,
-	PROGRESS_BOOT_ROM_PCIE_PHY_SRAM_INIT_TIMEDOUT = 102,
-	PROGRESS_BOOT_ROM_PCIE_PHY_PARA_STS_CLR_TIMEDOUT = 103,
-	PROGRESS_BOOT_ROM_PCIE_LINK_LIMIT_GEN1 = 104,
-	PROGRESS_BOOT_ROM_PCIE_LINK_IATU_CONFIG = 105,
-	PROGRESS_BOOT_ROM_PCIE_LINK_FINISH = 106,
-	PROGRESS_BOOT_ROM_D2D_LINK_START = 107,
-	PROGRESS_BOOT_ROM_D2D_LINK_STS_BUSY = 108,
-	PROGRESS_BOOT_ROM_D2D_LINK_INTER_DIE_SYNC = 109,
-	PROGRESS_BOOT_ROM_D2D_LINK_INIT_TIMEDOUT = 110,
-	PROGRESS_BOOT_ROM_D2D_LINK_INIT_SUCCESS = 111,
-	PROGRESS_BOOT_ROM_RECOVERY_END = 112,
-	PROGRESS_BOOT_ROM_IN_RECOVERY_IMG = 113,
-	PROGRESS_RECOVERY_IMG_IMG0 = 114,
-	PROGRESS_RECOVERY_IMG_IMG0_NO_IMG_DESC = 115,
-	PROGRESS_RECOVERY_IMG_IMG0_LOADADDR_INVALID = 116,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_STARTADDR_INVALID = 117,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_START = 118,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_CRC_FAILED = 119,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_UNALIGNED_DEST = 120,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_INVALID_IMG_SIZE = 121,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_UNALIGNED_IMG_SIZE = 122,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_INVALID_FLASH_OFFSET = 123,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_ERROR = 124,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_FINISH = 125,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_VERIFICATION_FAIL = 126,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_START = 127,
-	PROGRESS_BOOT_RECOVERY_IMG_IMG0_FINISH = 128,
-	PROGRESS_BOOT_INFO_IMG_DESC_INFO_SET_START = 129,
-	PROGRESS_BOOT_INFO_IMG_DESC_INFO_SET_FINISH = 130,
-	PROGRESS_BOOT_INFO_IMG_DESC_INFO_GET_START = 131,
-	PROGRESS_BOOT_INFO_IMG_DESC_INFO_GET_FINISH = 132,
-	PROGRESS_BOOT_INFO_LFS_DISK_MOUNT_FAIL = 133,
-	PROGRESS_BOOT_INFO_LFS_FILE_OPEN_CFG_FAIL = 134,
-	PROGRESS_BOOT_INFO_LFS_FILE_COPY_PART_FAIL = 135,
-	PROGRESS_BOOT_INFO_LFS_FILE_SET_FAIL = 136,
-	PROGRESS_BOOT_INFO_LFS_FILE_NAME_FAIL = 137,
-	PROGRESS_BOOT_INFO_LFS_FINISHED, /* TODO */
+	PROGRESS_INIT_EFUSE_SPI_IMG_DESC_FAIL = 39,
+	PROGRESS_INIT_EFUSE_SPI_EXCEEDS_MAX_SIZE = 40,
+	PROGRESS_INIT_EFUSE_SPI_DMA_COPY_FAIL = 41,
+	PROGRESS_INIT_EFUSE_SPI_MAGIC_FAIL = 42,
+	PROGRESS_INIT_EFUSE_SPI_DESC_COUNT_FAIL = 43,
+	PROGRESS_INIT_EFUSE_SPI_DECODE_FAIL = 44,
+	PROGRESS_INIT_EFUSE_SPI_SLV_TIMEDOUT = 45,
+	PROGRESS_REVOKE_LOC_ASIC_EFUSE = 46,
+	PROGRESS_REVOKE_LOC_FAKE_FUSE = 47,
+	PROGRESS_KEY_LOC = 48,
+	PROGRESS_KEY_LOC_0 = 49,
+	PROGRESS_KEY_LOC_1 = 50,
+	PROGRESS_VER_LOC = 51,
+	PROGRESS_VER_LOC_0 = 52,
+	PROGRESS_VER_LOC_1 = 53,
+	PROGRESS_SLV_DIE_WAITING_ON_RECOVERY_IND = 54,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_START = 55,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_VERSION_FAIL = 56,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_KEY_REVOKED = 57,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_KEY_HASH_FAIL = 58,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_AUTH_FAIL = 59,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_SUCCESS = 60,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_START = 61,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_VERSION_FAIL = 62,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_KEY_REVOKED = 63,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_KEY_HASH_FAIL = 64,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_AUTH_FAIL = 65,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_SUCCESS = 66,
+	PROGRESS_BRINGUP_PCI_PRI = 67,
+	PROGRESS_BRINGUP_PCI_PRI_NO_TMG_DESC = 68,
+	PROGRESS_BRINGUP_PCI_PRI_LOADADDR_INVALID = 69,
+	PROGRESS_BRINGUP_PCI_PRI_STARTADDR_INVALID = 70,
+	PROGRESS_BRINGUP_PCI_PRI_TOO_BIG = 71,
+	PROGRESS_BRINGUP_PCI_PRI_CRC_FAIL = 72,
+	PROGRESS_BRINGUP_PCI_PRI_UNALIGNED_DEST_ADDR = 73,
+	PROGRESS_BRINGUP_PCI_PRI_INVALID_IMG_SIZE = 74,
+	PROGRESS_BRINGUP_PCI_PRI_UNALIGNED_IMG_SIZE = 75,
+	PROGRESS_BRINGUP_PCI_PRI_INVALID_FLASH_OFFSET = 76,
+	PROGRESS_BRINGUP_PCI_PRI_DMA_ERROR = 77,
+	PROGRESS_BRINGUP_PCI_PRI_DMA = 78,
+	PROGRESS_BRINGUP_PCI_PRI_KEY_AUTH_FAIL = 79,
+	PROGRESS_BRINGUP_PCI_PRI_VERIFY_SUCCESS = 80,
+	PROGRESS_BRINGUP_PCI_SEC = 81,
+	PROGRESS_BRINGUP_PCI_SEC_NO_IMG_DESC = 82,
+	PROGRESS_BRINGUP_PCI_SEC_LOADADDR_INVALID = 83,
+	PROGRESS_BRINGUP_PCI_SEC_STARTADDR_INVALID = 84,
+	PROGRESS_BRINGUP_PCI_SEC_TOO_BIG = 85,
+	PROGRESS_BRINGUP_PCI_SEC_CRC_FAIL = 86,
+	PROGRESS_BRINGUP_PCI_SEC_UNALIGNED_DEST_ADDR = 87,
+	PROGRESS_BRINGUP_PCI_SEC_INVALID_IMG_SIZE = 88,
+	PROGRESS_BRINGUP_PCI_SEC_UNALIGNED_IMG_SIZE = 89,
+	PROGRESS_BRINGUP_PCI_SEC_INVALID_FLASH_OFFSET = 90,
+	PROGRESS_BRINGUP_PCI_SEC_DMA_ERROR = 91,
+	PROGRESS_BRINGUP_PCI_SEC_DMA = 92,
+	PROGRESS_BRINGUP_PCI_SEC_KEY_AUTH_FAIL = 93,
+	PROGRESS_BRINGUP_PCI_SEC_VERIFY_SUCCESS = 94,
+	PROGRESS_BOOT_ROM_TPM = 95,
+	PROGRESS_BOOT_ROM_TPM_INIT_FAIL = 96,
+	PROGRESS_BOOT_ROM_TPM_STARTUP_FAIL = 97,
+	PROGRESS_BOOT_ROM_SVN_TPM_PCR_EXTEND_FAIL = 98,
+	PROGRESS_BOOT_ROM_MANIFEST_TPM_PCR_EXTEND_FAIL = 99,
+ 	PROGRESS_BOOT_ROM_TPM_FSM_OUT_OF_SYNC = 100,
+	PROGRESS_BOOT_ROM_TPM_FINISH = 101,
+	PROGRESS_PPBOOT_START = 102,
+	PROGRESS_PPBOOT_FINISH = 103,
+	PROGRESS_BOOT_ROM_IN_RECOVERY = 104,
+	PROGRESS_BOOT_ROM_IN_ROM_RECOVERY = 105,
+	PROGRESS_BOOT_ROM_PCIE_LINK_START = 106,
+	PROGRESS_BOOT_ROM_PCIE_LINK_INIT = 107,
+	PROGRESS_BOOT_ROM_PCIE_PHY_SRAM_INIT_TIMEDOUT = 108,
+	PROGRESS_BOOT_ROM_PCIE_PHY_PARA_STS_CLR_TIMEDOUT = 109,
+	PROGRESS_BOOT_ROM_PCIE_LINK_LIMIT_GEN1 = 110,
+	PROGRESS_BOOT_ROM_PCIE_LINK_IATU_CONFIG = 111,
+	PROGRESS_BOOT_ROM_PCIE_LINK_FINISH = 112,
+	PROGRESS_BOOT_ROM_D2D_LINK_START = 113,
+	PROGRESS_BOOT_ROM_D2D_LINK_STS_BUSY = 114,
+	PROGRESS_BOOT_ROM_D2D_LINK_INTER_DIE_SYNC = 115,
+	PROGRESS_BOOT_ROM_D2D_LINK_INIT_TIMEDOUT = 116,
+	PROGRESS_BOOT_ROM_D2D_LINK_INIT_SUCCESS = 117,
+	PROGRESS_BOOT_ROM_RECOVERY_END = 118,
+	PROGRESS_BOOT_ROM_IN_RECOVERY_IMG = 119,
+	PROGRESS_RECOVERY_IMG_IMG0 = 120,
+	PROGRESS_RECOVERY_IMG_IMG0_NO_IMG_DESC = 121,
+	PROGRESS_RECOVERY_IMG_IMG0_LOADADDR_INVALID = 122,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_STARTADDR_INVALID = 123,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_START = 124,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_CRC_FAILED = 125,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_UNALIGNED_DEST = 126,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_INVALID_IMG_SIZE = 127,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_UNALIGNED_IMG_SIZE = 128,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_INVALID_FLASH_OFFSET = 129,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_ERROR = 130,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_FINISH = 131,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_VERIFICATION_FAIL = 132,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_START = 133,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_FINISH = 134,
+	PROGRESS_BOOT_INFO_IMG_DESC_INFO_SET_START = 135,
+	PROGRESS_BOOT_INFO_IMG_DESC_INFO_SET_FINISH = 136,
+	PROGRESS_BOOT_INFO_IMG_DESC_INFO_GET_START = 137,
+	PROGRESS_BOOT_INFO_IMG_DESC_INFO_GET_FINISH = 138,
+	PROGRESS_BOOT_INFO_LFS_DISK_MOUNT_FAIL = 139,
+	PROGRESS_BOOT_INFO_LFS_FILE_OPEN_CFG_FAIL = 140,
+	PROGRESS_BOOT_INFO_LFS_FILE_COPY_PART_FAIL = 141,
+	PROGRESS_BOOT_INFO_LFS_FILE_SET_FAIL = 142,
+	PROGRESS_BOOT_INFO_LFS_FILE_NAME_FAIL = 143,
+	PROGRESS_BOOT_INFO_LFS_FINISHED = 144,
 };
 
 #endif /* _PROGRESS_BITS_H_ */
